marklogic-data-hub/build.gradle

```diff
@@ -32,17 +35,32 @@ ext.junitPlatformVersion = '1.4.0'
 ext.junitJupiterVersion  = '5.4.0'
 
 dependencies {
-    compile 'com.marklogic:marklogic-client-api:4.2.0'
-    compile('com.marklogic:ml-app-deployer:3.16.3'){
+    // Forcing use of 1.3.71 - which is what OkHttp 4.7.2 wants, which is what Java Client 5.3 uses - so that 1.2.x
+    // versions don't come onto the classpath
+    compile "org.jetbrains.kotlin:kotlin-stdlib:1.3.71"
+    compile "org.jetbrains.kotlin:kotlin-stdlib-common:1.3.71"
+    compile ('com.marklogic:marklogic-client-api:5.3.2'){
+        exclude group: 'com.fasterxml.woodstox', module: 'woodstox-core'
+        // Oddly, the OkHttp 4.4.0 dependency includes version 1.2.70 and 1.3.x of these dependencies. This causes
+        // class conflict errors. So they're excluded from here, and we force inclusion of 1.3.x below.
+        exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib'
+        exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib-common'
+    }
+    compile('com.marklogic:ml-app-deployer:4.1.0'){
```

Comment:
This still needs to be 3.16.3 - can't go to 4.x because it requires Java 9. Be sure to test out the DHF Gradle plugin on one of the example projects after making the changes here, and be sure to run Java 8, as that's what DHF 5.2.x supports.